--- config-2_1.yml	2022-10-05 15:05:36.956462247 +0100
+++ config-2_1.yml.MIDRES	2022-10-05 15:07:24.792794712 +0100
@@ -160,14 +160,14 @@
 j8_par_executor: &j8_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j8_small_par_executor: &j8_small_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j8_small_executor: &j8_small_executor
   executor:
@@ -175,29 +175,41 @@
     exec_resource_class: medium
   parallelism: 1
 
+j8_large_par_executor: &j8_large_par_executor
+  executor:
+    name: java8-executor
+    exec_resource_class: large
+  parallelism: 50
+
+j8_very_large_par_executor: &j8_very_large_par_executor
+  executor:
+    name: java8-executor
+    exec_resource_class: xlarge
+  parallelism: 100
+
 j8_medium_par_executor: &j8_medium_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 4
 
 j8_seq_executor: &j8_seq_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j11_par_executor: &j11_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_small_par_executor: &j11_small_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j11_small_executor: &j11_small_executor
   executor:
@@ -205,35 +217,47 @@
     #exec_resource_class: medium
   parallelism: 1
 
+j11_large_par_executor: &j11_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 50
+
 j8_repeated_utest_executor: &j8_repeated_utest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j8_repeated_dtest_executor: &j8_repeated_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j8_repeated_upgrade_dtest_executor: &j8_repeated_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 25
 
 j8_repeated_jvm_upgrade_dtest_executor: &j8_repeated_jvm_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j11_repeated_utest_executor: &j11_repeated_utest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_repeated_dtest_executor: &j11_repeated_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j8_with_dtests_jobs: &j8_with_dtests_jobs
   jobs:
@@ -1044,7 +1068,7 @@
           target: testclasslist-system-keyspace-directory
 
   j8_dtests-with-vnodes:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1058,7 +1082,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j11_dtests-with-vnodes:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1073,7 +1097,7 @@
         pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j8_dtests-no-vnodes:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1087,7 +1111,7 @@
           pytest_extra_args: '--skip-resource-intensive-tests'
 
   j11_dtests-no-vnodes:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1102,7 +1126,7 @@
         pytest_extra_args: '--skip-resource-intensive-tests'
 
   j8_upgradetests-no-vnodes:
-    <<: *j8_par_executor
+    <<: *j8_very_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1116,7 +1140,7 @@
           pytest_extra_args: '--execute-upgrade-tests-only --upgrade-target-version-only --upgrade-version-selection all'
 
   j8_cqlsh-dtests-py3-with-vnodes:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1131,7 +1155,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh-dtests-py38-with-vnodes:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1149,7 +1173,7 @@
           python_version: '3.8'
 
   j8_cqlsh-dtests-py3-no-vnodes:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1164,7 +1188,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh-dtests-py38-no-vnodes:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1182,7 +1206,7 @@
           python_version: '3.8'
 
   j11_cqlsh-dtests-py3-with-vnodes:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1197,7 +1221,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh-dtests-py38-with-vnodes:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1215,7 +1239,7 @@
           python_version: '3.8'
 
   j11_cqlsh-dtests-py3-no-vnodes:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1230,7 +1254,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh-dtests-py38-no-vnodes:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
